test/stampit-specs.js

```diff
@@ -93,70 +93,136 @@ test('stampit().methods(a, b)', function () {
     'Should mixIn objects when multiple methods are passed.');
 });
 
-module('Basics State');
+module('Basics Props');
 
-test('stampit({}, state)', function () {
+test('stampit({}, props)', function () {
```

Comment:
Signature is `stampit(methods, refs, enclose, props)`. You have `props` and `refs` swapped in the tests.